tensorboard/data/server/lib.rs

```diff
@@ -19,3 +19,12 @@ pub mod masked_crc;
 
 #[cfg(test)]
 mod scripted_reader;
+
+/// Protocol buffer bindings.
+#[allow(clippy::all)]
```

Comment:
:) What does this do? Is this running clippy on the proto definitions?

Reply:
The opposite: it _suppresses_ all Clippy lint warnings inside the
generated code. There are a few different [diagnostic attributes][1]:

```rust
#[allow(dead_code)]
#[warn(dead_code)]
#[deny(dead_code)]
#[forbid(dead_code)]
```

where `deny` and `forbid` are both hard errors, but `forbid` also
prevents you from `allow`ing it in any child item. So, if you see a
crate that has `#![forbid(unsafe)]` at the top, then you’re immediately
100% sure that it has no uses of `unsafe`.

Here, `clippy::all` is a… shortcut? alias? group? (don’t know the right
term) for “all the Clippy lints”, so this is “allow any Clippy lint
without erroring or warning about it”.

Context: <https://github.com/danburkert/prost/issues/379>

[1]: https://doc.rust-lang.org/reference/attributes/diagnostics.html#lint-check-attributes